tests/test_modeling_deberta.py

```diff
@@ -0,0 +1,269 @@
+# coding=utf-8
+# Copyright 2018 Microsoft Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers import is_torch_available
+from transformers.testing_utils import require_torch, torch_device
+
+from .test_configuration_common import ConfigTester
+from .test_modeling_common import ModelTesterMixin, ids_tensor
+
+
+if is_torch_available():
+    from transformers import (
+        DeBERTaConfig,
+        DeBERTaModel,
+        # XxxForMaskedLM,
+        # XxxForQuestionAnswering,
+        DeBERTaForSequenceClassification,
+        # XxxForTokenClassification,
+    )
+    from transformers.modeling_deberta import DeBERTa_PRETRAINED_MODEL_ARCHIVE_LIST
+
+
+@require_torch
+class DeBERTaModelTest(ModelTesterMixin, unittest.TestCase):
+
+    all_model_classes = (
+        (
+            DeBERTaModel,
+            DeBERTaForSequenceClassification,
+        )  # , DeBERTaForMaskedLM, DeBERTaForQuestionAnswering, DeBERTaForTokenClassification)
+        if is_torch_available()
+        else ()
+    )
+
+    test_torchscript = False
+    test_pruning = False
+    test_resize_embeddings = False
+    test_head_masking = False
+    test_missing_keys = False
+    is_encoder_decoder = False
+
+    class DeBERTaModelTester(object):
```

Comment:
this doesn't need to be indented it can be a top level objected on the file, or imported from another test file.